cmd/dep/status_test.go

```diff
@@ -407,23 +407,15 @@ func TestCollectConstraints(t *testing.T) {
 	h := test.NewHelper(t)
 	defer h.Cleanup()
 
-	h.TempDir("src")
-	pwd := h.Path(".")
-	h.TempFile(filepath.Join("src", "dep.go"), `
-		package dep
-		import (
-			_ "github.com/boltdb/bolt"
-			_ "github.com/sdboyer/deptest"
-			_ "github.com/sdboyer/dep-test"
-			_ "github.com/sdboyer/deptestdos"
-		)
-		type FooBar int
-	`)
+	testdir := filepath.Join("src", "collect_constraints_test")
+	h.TempDir(testdir)
+	h.TempCopy(filepath.Join(testdir, "main.go"), "status/collect_constraints/main.go")
```

Comment:
Maybe use `filepath.Join()` here for `"status/collect_constraints/main.go"` to be safe?